### ISSUE TITLE
Installation runs into further dependency issues

### ISSUE THREAD

**Issue by bdzyubak-aiq** [ichxw/cuda_11.7_installation_on_Ubuntu_22.04#1]
Thanks a lot for working on this installation script. Installing CUDA on Linux is a huge pain. However, using the long list of wget's to install dependencies, I run into further dependency issues: 

dependency problems prevent configuration of cuda-tools-11-7:
 cuda-tools-11-7 depends on cuda-command-line-tools-11-7
 
sudo apt-get install cuda-command-line-tools-11-7 yields many more nested layers

Were you ever able to solve this?

**Comment by ichxw** [comment]
Yes. That was a summary after a successful installation. The tip in this instruction was to download any missing packages from https://developer.download.nvidia.com/compute/cuda/repos/$distro/$arch instead of using sudo apt-get for installation. 
```
The tip is to follow the error message and download whatever is missing from https://developer.download.nvidia.com/compute/cuda/repos/$distro/$arch and install it.
```

**Comment by bdzyubak-aiq** [comment]
That didn't work for me. Fortunately, for anyone still looking for solutions, using the runfile installer off the CUDA website worked

sudo apt-get purge nvidia*
sudo apt remove nvidia-*
sudo rm /etc/apt/sources.list.d/cuda*
sudo apt-get autoremove && sudo apt-get autoclean
sudo rm -rf /usr/local/cuda*

Reboot

![image](https://github.com/ichxw/cuda_11.7_installation_on_Ubuntu_22.04/assets/119627847/94c44fe0-40cf-4022-847f-cd89e0debf3d)
